pvcviewer-controller/rockcraft.yaml

```diff
@@ -0,0 +1,45 @@
+# Source https://github.com/kubeflow/kubeflow/blob/v1.9.0/components/pvcviewer-controller/Dockerfile
+name: pvcviewer-controller
+summary: An image for PVCViewer
+description: |
+  This image is used as part of the Charmed Kubeflow product. 
+  PVCViewers enable users to open a filebrowser on arbitrary persistent volume claims, letting them inspect, download, upload and manipulate data.
+version: "1.9.0"
+license: Apache-2.0
+base: ubuntu@22.04
+
+platforms:
+  amd64:
+
+services:
+  pvcviewer-controller:
+    override: merge
+    command: "pvc-viewer"
+    startup: enabled
+    user: ubuntu
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  pvcviewer-controller:
```

Comment:
To stay consistent I recomment building like this https://github.com/canonical/kubeflow-rocks/blob/main/admission-webhook/rockcraft.yaml#L41 with proper environment